features/authentication-framework/org.wso2.carbon.identity.application.authentication.framework.server.feature/resources/application-authentication.xml

```diff
@@ -130,17 +130,17 @@
         <AuthenticatorConfig name="EmailOTP" enabled="true">
             <Parameter name="GmailClientId">gmailClientIdValue</Parameter>
             <Parameter name="GmailClientSecret">gmailClientSecretValue</Parameter>
-            <Parameter name="SendgridAPIKey">sendgridAPIKeyValue</Parameter>
             <Parameter name="GmailRefreshToken">gmailRefreshTokenValue</Parameter>
             <Parameter name="GmailEmailEndpoint">https://www.googleapis.com/gmail/v1/users/[userId]/messages/send</Parameter>
+            <Parameter name="GmailAuthTokenType">Bearer</Parameter>
+            <Parameter name="GmailTokenEndpoint">https://www.googleapis.com/oauth2/v3/token</Parameter>
+            <Parameter name="SendgridAPIKey">sendgridAPIKeyValue</Parameter>
             <Parameter name="SendgridEmailEndpoint">https://api.sendgrid.com/api/mail.send.json</Parameter>
-            <Parameter name="accessTokenRequiredAPIs">Gmail</Parameter>
-            <Parameter name="apiKeyHeaderRequiredAPIs">Sendgrid</Parameter>
             <Parameter name="SendgridFormData">sendgridFormDataValue</Parameter>
             <Parameter name="SendgridURLParams">sendgridURLParamsValue</Parameter>
-            <Parameter name="GmailAuthTokenType">Bearer</Parameter>
-            <Parameter name="GmailTokenEndpoint">https://www.googleapis.com/oauth2/v3/token</Parameter>
+            <Parameter name="accessTokenRequiredAPIs">Gmail</Parameter>
             <Parameter name="SendgridAuthTokenType">Bearer</Parameter>
+            <Parameter name="apiKeyHeaderRequiredAPIs">Sendgrid</Parameter>
```

Comment:
Better to keep only gmail settings enabled and comment out sendgrid related configs with a proper comment